clang/include/clang/Driver/Options.td

```diff
@@ -9111,3 +9118,50 @@ def wasm_opt : Flag<["--"], "wasm-opt">,
   Group<m_Group>,
   HelpText<"Enable the wasm-opt optimizer (default)">,
   MarshallingInfoNegativeFlag<LangOpts<"NoWasmOpt">>;
+
+
+
+//===----------------------------------------------------------------------===//
+// cl2000 Options
+//===----------------------------------------------------------------------===//
+
+
+
+
+def cl2000_include_path : Joined<["--"], "include_path=">, Group<cl2000_group>,
+                    HelpText<"specify include search paths for cl2000 driver mode">, Alias<isystem>;
+
+def eabi :        Joined<["--"], "abi=">, Group<cl2000_group>,
```

Comment:
I am concerned of adding driver options (perhaps  mostly ignored). A better way is to change language servers and clang-tidy to ignore options. These tools can simply set an environment variable

```
CCC_OVERRIDE_OPTIONS=#x-fxxx clang -fsyntax-only -fxxx /tmp/t/a.c   # does not error on -fxxx
```